sdk/datafactory/Azure.ResourceManager.DataFactory/samples/Generated/Samples/Sample_DataFactoryIntegrationRuntimeResource.cs

```diff
@@ -606,7 +606,7 @@ public async Task UpdateIntegrationRuntimeNode_IntegrationRuntimeNodesUpdate()
         public async Task GetIPAddressIntegrationRuntimeNode_IntegrationRuntimeNodesGetIpAddress()
         {
             // Generated from example definition: specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/examples/IntegrationRuntimeNodes_GetIpAddress.json
-            // this example is just showing the usage of "IntegrationRuntimeNodes_GetIPAddress" operation, for the dependent resources, they will have to be created separately.
+            // this example is just showing the usage of "IntegrationRuntimeNodes_GetIpAddress" operation, for the dependent resources, they will have to be created separately.
```

Comment:
it's `Ip`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/datafactory.json#L1724